modules/persistent-storage-csi-example-deployment.adoc

```diff
@@ -0,0 +1,261 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-csi.adoc
+
+[id="csi-example-deployment-{context}"]
+= Example CSI deployment
+
+Since {product-title} does not ship with any CSI driver installed, this 
+example shows how to deploy a community driver for OpenStack Cinder in 
+{product-title}.
+
+.Procedure
+
+. Create a new project where the CSI components will run and a new service 
+account that will run the components. Explicit node selector is used to 
```

Comment:
account will run the components? 

An explicit node selector?

Reply:
I've adjusted the first sentence to be:

    Create a new project where the CSI components will run, and then create a new service account to run these components.

I've implemented the second suggestion.